src/Microsoft.DotNet.Interactive.SqlServer/SqlToolsServiceDiscovery.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.DotNet.Interactive.SqlServer
+{
+    internal static class SqlToolsServiceDiscovery
+    {
+        public static ResolvedPackageReference FindResolvedPackageReference(this Kernel rootKernel)
+        {
+            var runtimePackageIdSuffix = "native.Microsoft.SqlToolsService";
+            foreach (var kernel in rootKernel.SubkernelsAndSelf().OfType<ISupportNuget>())
```

Comment:
This whole block can be a `return ... SingleOrDefault`

Reply:
ok

Reply:
Fsharp could load the package too, We need to make sure we find the one that has the package

Reply:
`FirstOrDefault` then?